TGBotFramework/BotFramework/Attributes/MessageAttribute.cs

```diff
@@ -88,36 +86,37 @@ private bool CanHandleMessage(Message message)
-                    MessageFlag.HasAudio        => message.Audio != null,
-                    MessageFlag.HasVideo        => message.Video != null,
-                    MessageFlag.HasDocument     => message.Document != null,
-                    MessageFlag.HasAnimation    => message.Animation != null,
-                    MessageFlag.HasGame         => message.Game != null,
-                    MessageFlag.HasCaption      => !string.IsNullOrEmpty(message.Caption),
-                    MessageFlag.HasPhoto        => message.Photo != null,
-                    MessageFlag.HasSticker      => message.Sticker != null,
-                    MessageFlag.HasVideoNote    => message.VideoNote != null,
-                    MessageFlag.HasContact      => message.Contact != null,
-                    MessageFlag.HasLocation     => message.Location != null,
-                    MessageFlag.HasVenue        => message.Venue != null,
-                    MessageFlag.HasPoll         => message.Poll != null,
-                    MessageFlag.HasDice         => message.Dice != null,
-                    MessageFlag.HasKeyboard     => message.ReplyMarkup != null,
-                    _ => false
-                };
+                    {
+                        MessageFlag.HasForward => message.ForwardFrom != null || message.ForwardFromChat != null,
+                        MessageFlag.IsReply => message.ReplyToMessage != null,
+                        MessageFlag.HasText => !string.IsNullOrEmpty(message.Text),
+                        MessageFlag.HasEntity => message.Entities != null || message.CaptionEntities != null,
+                        MessageFlag.HasVoice => message.Voice != null,
+                        MessageFlag.HasAudio => message.Audio != null,
+                        MessageFlag.HasVideo => message.Video != null,
+                        MessageFlag.HasDocument => message.Document != null,
+                        MessageFlag.HasAnimation => message.Animation != null,
+                        MessageFlag.HasGame => message.Game != null,
+                        MessageFlag.HasCaption => !string.IsNullOrEmpty(message.Caption),
+                        MessageFlag.HasPhoto => message.Photo != null,
+                        MessageFlag.HasSticker => message.Sticker != null,
+                        MessageFlag.HasVideoNote => message.VideoNote != null,
+                        MessageFlag.HasContact => message.Contact != null,
+                        MessageFlag.HasLocation => message.Location != null,
+                        MessageFlag.HasVenue => message.Venue != null,
+                        MessageFlag.HasPoll => message.Poll != null,
+                        MessageFlag.HasDice => message.Dice != null,
+                        MessageFlag.HasKeyboard => message.ReplyMarkup != null,
+                        MessageFlag.HasNewChatMembers => message.NewChatMembers != null && message.NewChatMembers?.Length != 0,
```

Comment:
Now you are double checking for null:) 
`MessageFlag.HasNewChatMembers => message.NewChatMembers?.Length != 0` is enough
Or you can use [Any](https://docs.microsoft.com/ru-ru/dotnet/api/system.linq.enumerable.any?view=net-5.0):
`MessageFlag.HasNewChatMembers => message.NewChatMembers?.Any() == true`